tests/unit/schema/test_api.py

```diff
@@ -393,6 +424,21 @@ def test_services():
     assert method.input.meta.doc == 'This is the GetFooRequest message.'
     assert method.output.name == 'GetFooResponse'
     assert method.output.meta.doc == 'This is the GetFooResponse message.'
+    assert method.timeout - 30.0 < 5e-10
```

Comment:
Nit: why not lift the `- x < 5e-19` into an approx_eq function? Makes it more correct, too.

```python
def approx_eq(x, y, epsilon=5e-19):
   return abs(x - y) < epsilon
```

Reply:
Replacing with `pytest.approx`. I meant to look up the syntax and forgot.